app/misc/authenticate.ts

```diff
@@ -502,3 +431,10 @@ function displayIssues() {
           });
       }
     }
+function getUsername() {
+  if (!account) {
+    return null;
```

Comment:
Prompt the user if this catches something like "could not get account, try signing in again"